package.json

```diff
@@ -110,6 +109,10 @@
     "webpack-hasjs-plugin": "^1.0.3",
     "webpack-merge": "^4.2.2"
   },
+  "overrides": {
```

Comment:
@ziveo Why are the overrides needed?

Reply:
Libraries that are using old versions of `node-forge` and `glob-parent` are not updated for long time and overrides is way to force installations of libraries that dont have security vulnerability